### PR TITLE
Add a very basic `FallbackMailbox`

### PR DESCRIPTION
**Pull request by sampoder**
## Summary of the problem

https://heroku-app97991095.airbrake.io/projects/288439/groups/3687002297321877959

## Describe your changes

Basically, if routing fails... we tell them an unknown error occurred.

### CODE REVIEW

**Comment by sampoder** [comment]
Ok, now it forwards!

File: app/mailboxes/fallback_mailbox.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class FallbackMailbox < ApplicationMailbox
+  # mail --> Mail object, this actual email
+  # inbound_email => ActionMailbox::InboundEmail record --> the active storage record
+
+  def process
+    bounce_with ForwardMailer.forward(mail:)
```
Comment (sampoder) [inline app/mailboxes/fallback_mailbox.rb]:
```suggestion
    bounce_with ForwardMailer.forward(incoming_mail: mail)
```

File: app/mailers/forward_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ForwardMailer < ApplicationMailer
+  def forward(mail:)
+    mail to: "hcb@hackclub.com", subject: "Fwd: #{mail.subject}", content: mail.content
+  end
```
Comment (sampoder) [inline app/mailers/forward_mailer.rb]:
```suggestion
  def forward(incoming_mail:)
    mail to: "hcb@hackclub.com", subject: "Fwd: #{incoming_mail.subject}", content: incoming_mail.content
  end
```

**Review by sarthaktexas — APPROVED** [review]

File: app/mailers/forward_mailer.rb
Comment (garyhtou) [inline app/mailers/forward_mailer.rb:1]: Let's rename this to `MailboxMailer`

File: app/mailers/forward_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ForwardMailer < ApplicationMailer
+  def forward(incoming_mail:)
+    mail to: "hcb@hackclub.com", subject: "Fwd: #{incoming_mail.subject}", content: incoming_mail.content
```
Comment (garyhtou) [inline app/mailers/forward_mailer.rb]:
```suggestion
  def forward(incoming_mail:, to:)
    mail to:, subject: "Fwd: #{incoming_mail.subject}", content: incoming_mail.content
```

Comment (garyhtou) [inline app/mailers/forward_mailer.rb]: I'm assuming when we forward in this way, we lose a lot of the context such as the original `from` email address.

Should we add the `from` email address as the `reply-to`?

File: app/mailboxes/fallback_mailbox.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class FallbackMailbox < ApplicationMailbox
+  # mail --> Mail object, this actual email
+  # inbound_email => ActionMailbox::InboundEmail record --> the active storage record
+
+  def process
+    bounce_with ForwardMailer.forward(incoming_mail: mail)
```
Comment (sampoder) [inline app/mailboxes/fallback_mailbox.rb]:
```suggestion
    bounce_with ForwardMailer.forward(incoming_mail: mail, to: "hcb@hackclub.com")
```

File: app/mailers/mailbox_mailer.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class MailboxMailer < ApplicationMailer
+  def forward(incoming_mail:, to:)
+    mail to:, subject: "Fwd: #{incoming_mail.subject}", content: incoming_mail.content
```
Comment (sampoder) [inline app/mailers/mailbox_mailer.rb]:
```suggestion
    mail to:, subject: "Fwd: #{incoming_mail.subject}", reply_to: incoming_mail.to, content: incoming_mail.content
```

File: app/mailers/mailbox_mailer.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class MailboxMailer < ApplicationMailer
+  def forward(incoming_mail:, to:)
+    mail to:, subject: "Fwd: #{incoming_mail.subject}", reply_to: incoming_mail.to, content: incoming_mail.content
```
Comment (sampoder) [inline app/mailers/mailbox_mailer.rb]:
```suggestion
    mail to:, subject: "Fwd: #{incoming_mail.subject} (#{incoming_mail.to.first})", reply_to: incoming_mail.from, content: incoming_mail.content
```

**Review by garyhtou — APPROVED** [review]

**Comment by garyhtou** [comment]
Let's post a message in `#hcb-team` so that ops is aware

**Pull request merged**